geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -1042,6 +1042,10 @@ public boolean initialImagePut(final Object key, final long lastModified, Object
           } finally {
             if (done && result) {
               initialImagePutEntry(newRe);
+              if (owner instanceof BucketRegionQueue) {
```

Comment:
Why is this needed here?  Are we missing events currently?

Reply:
Yes. This is GII logic. I talked to you last Friday.